.github/workflows/hash-backend-cd.yml

```
@@ -13,85 +14,105 @@ env:
   AWS_REGION: ${{ secrets.AWS_REGION }}
   AWS_ECR_URL: ${{ secrets.AWS_ECR_URL }}
 
-  # Format "user:pass"
-  HASH_POSTGRES_MIGRATE_USER_PASS: ${{ secrets.HASH_POSTGRES_MIGRATE_USER_PASS }}
-  # Format: "env.id.region.rds.amazonaws.com/database"
-  HASH_POSTGRES_HOST_DB: ${{ secrets.HASH_POSTGRES_HOST_DB }}
-
   GH_RUN_ID: ${{ github.run_id }}
 
   HASH_API_RESOURCE_NAME: ${{ secrets.HASH_API_RESOURCE_NAME }}
-  HASH_REALTIME_RESOURCE_NAME: ${{ secrets.HASH_REALTIME_RESOURCE_NAME }}
-  HASH_SEARCHLOADER_RESOURCE_NAME: ${{ secrets.HASH_SEARCHLOADER_RESOURCE_NAME }}
+  HASH_GRAPH_RESOURCE_NAME: ${{ secrets.HASH_GRAPH_RESOURCE_NAME }}
+  HASH_KRATOS_RESOURCE_NAME: ${{ secrets.HASH_KRATOS_RESOURCE_NAME }}
 
   HASH_ECS_CLUSTER_NAME: ${{ secrets.HASH_ECS_CLUSTER_NAME }}
+  HASH_APP_SERVICE_NAME: ${{ secrets.HASH_APP_SERVICE_NAME }}
+
+  # To be uncommented once we migrate in the job below
+  # HASH_GRAPH_PG_USER: ${{ secrets.HASH_GRAPH_PG_USER }}
+  # HASH_GRAPH_PG_PASSWORD: ${{ secrets.HASH_GRAPH_PG_PASSWORD }}
+  # HASH_GRAPH_PG_HOST: ${{ secrets.HASH_GRAPH_PG_HOST }}
+  # HASH_GRAPH_PG_PORT: ${{ secrets.HASH_GRAPH_PG_PORT }}
+  # HASH_GRAPH_PG_DATABASE: ${{ secrets.HASH_GRAPH_PG_DATABASE }}
```

Reeeeally minor: it’s helpful to distinguish comments from commented-out lines of code by using `##` and `#`. This makes it possible to select a bunch of lines in an idea, use ‘toggle comment‘ shortcut and end up with a valid shell / yaml / etc. Not suggesting to make a change now, just sharing a side thought.

```suggestion
  ## To be uncommented once we migrate in the job below
  # HASH_GRAPH_PG_USER: ${{ secrets.HASH_GRAPH_PG_USER }}
  # HASH_GRAPH_PG_PASSWORD: ${{ secrets.HASH_GRAPH_PG_PASSWORD }}
  # HASH_GRAPH_PG_HOST: ${{ secrets.HASH_GRAPH_PG_HOST }}
  # HASH_GRAPH_PG_PORT: ${{ secrets.HASH_GRAPH_PG_PORT }}
  # HASH_GRAPH_PG_DATABASE: ${{ secrets.HASH_GRAPH_PG_DATABASE }}
```

---

I definitely think this makes sense, but given I'll be removing that line in #1877, I think it's fine to skip for now :)